python_apps/media-monitor/bin/airtime-media-monitor

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+exec 2>&1
+
+cd /usr/lib/airtime/media-monitor/
+
+set +e
+cat /etc/default/locale | grep -i "LANG=.*UTF-\?8"
+set -e
+if [ "$?" != "0" ]; then
+    echo "non UTF-8 default locale found in /etc/default/locale." > /var/log/airtime/media-monitor/error.log
+    exit 1
+fi
+
+export PYTHONPATH=/usr/lib/airtime/:/usr/lib/airtime/media-monitor/airtimefilemonitor/:\
```

Comment:
Can we refactor media-monitor to not need this custom python path? It needlessly complicates things.